### DISCUSSION TITLE
How can I change my username?

### DISCUSSION THREAD

**Discussion by unreliableart** [codesandbox/codesandbox-client#5531] [General] [answered]
I need to change my username, because when I signed in with google it gave me a default name. And, as you can see, my privacy is now *poof!* gone, because it's my real name. PLEASE HELP!

**Comment by unreliableart** [comment]
Wait what? My name changed? okay... creepy

**Comment by smhutch** [comment, marked as answer]
@unreliableart sorry that this didn't work as expected. We recently added support for changing both your account name and username. To do so:

1. Visit `https://codesandbox.io/u/YOUR_USERNAME`
2. Click "Edit Profile"
3. Update, and save
